src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuefinaldecision/IssueFinalDecisionSubmittedHandlerTest.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.issuefinaldecision;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.*;
```

Comment:
Swap all Assert methods for Assertions library

Reply:
done